component/grant-type/src/main/java/org/wso2/carbon/identity/oauth2/grant/jwt/JWTBearerGrantHandler.java

```diff
@@ -96,6 +96,8 @@
     private static final String OIDC_IDP_ENTITY_ID = "IdPEntityId";
     private static final String ERROR_GET_RESIDENT_IDP =
             "Error while getting Resident Identity Provider of '%s' tenant.";
+    private static final String ENFORCE_CERTIFICATE_VALIDITY
```

Comment:
do we need this config for public branch also? @senthalan @mefarazath

Reply:
There might be external IDPs who we do not have control over. So we decide to keep this option to the master as well